spec/lucky_cli/process_runner_spec.cr

```diff
@@ -54,12 +54,4 @@ describe LuckyCli::ProcessRunner do
     InstalledRunner.new.installed?.should be_true
     MissingRunner.new.installed?.should be_false
   end
-
-  it "can find an installed process runner and start it" do
```

Comment:
Removed this spec since it was using the actually ProcessRunners and since Nox is the first one, it was using that. The nox runner doesn't use the FakeProcess so it would have hung indefinitely.